### DISCUSSION TITLE
Social Media Icons

### DISCUSSION THREAD

**Discussion by shubhechchhakarmacharya** [tailwindlabs/heroicons#1088] [Ideas]
The absence of social network icons in Heroicons is unacceptable. It is also necessary for us as developers to not have to search other websites for icons in order to utilise them on websites belonging to several businesses.

**Comment by RoshanDasan** [comment]
It is indeed true. The experience with Heroicons is impressive. However, when it comes to social media icons, their absence could pose a significant challenge for developers.

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
